Cornelica/scenario/007_01.txt

```diff
@@ -24,7 +24,7 @@
 @bs f=アル@普通 body=-1 face=0 x=-350 y=50 t=200
 @bs f=リリィ@笑顔 body=-1 face=0 x=350 y=50 t=200
 
-[Lily] "The food is also delicious, so many people come for food too"
+[Lily]"The food is also delicious, so many people come for to eat too"
```

Comment:
`come for to eat too`? - Grammar-wise, this make no sense. At least to me. Maybe just dropping `for` might work.

Reply:
Copy-pasta madness, probably i was daydreaming @.@
Yeah... it's sound better